cmd/atlascmd/migrate.go

```diff
@@ -430,6 +430,7 @@ const (
 	formatGoose         = "goose"
 	formatFlyway        = "flyway"
 	formatLiquibase     = "liquibase"
+	formatDbmate        = "dbmate"
```

Comment:
```suggestion
	formatDBMate        = "dbmate"
```
Although the project uses `Dbmate`, I prefer to follow the [Go initialisms rule](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). @masseelch, your thoughts?

Reply:
Agree to that. Reviewing now.

Reply:
But the author of `dbmate` seems to describe it as `Dbmate`, so I believe Dbmate is an intrinsic nouns.
https://github.com/amacneil/dbmate#dbmate

Reply:
I know. The project name is `Dbmate`, but I prefer the variable name to be `DBMate`. Sorry for being annoying 😉 

Would you mind changing it to `DBMate`? Thanks